client/src/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,44 @@
+import { Box, Button, Grid } from '@material-ui/core';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+
+import LogoPic from '../../Images/logoPic.png';
+import userImage from '../../Images/userImage1.png';
+
+import { User } from '../../interface/User'; //temp
+
+export default function DashboardHeader(): JSX.Element {
+  const classes = useStyles();
+
+  const loggedInUser: User = { email: 'rose@gmail.com', username: 'Rose' }; //temp
+
+  return (
+    <Grid container item xs={12} direction="row" justify="center" alignItems="center" className={classes.navBar}>
+      <Box flexGrow={1}>
+        <Link to="/">
+          <img src={LogoPic} alt="Logo" className={classes.logo} />
+        </Link>
+      </Box>
+      <Grid item>
+        <Button className={`${classes.navBarItem} ${classes.navbarItemActive}`}>Shopping Lists</Button>
+        <Button className={classes.navBarItem}>Friends</Button>
+        <Button className={classes.navBarItem}>Notifications</Button>
```

Comment:
(bonus) you can make these links so that the navigation functionality is there, obviously they will redirect to nothing at the moment but it mean this logic won't need to be added later

Reply:
I did this but I was not sure if the notifications should have its own route. Because in the mockups it seems like its kinda like a modal that gets displayed when the button is clicked